app/helpers/alerts_helper.rb

```diff
@@ -1,3 +1,11 @@
 # frozen_string_literal: true
 module AlertsHelper
+  def alert_display
+    if alert.resolvable? == true && alert.resolved == true then
```

Comment:
Well, thinking more about it, I see both the branches having a common condition. How about rewriting the method as

```ruby
def alert_display(alert)
  return unless alert.resolvable?
  return '✓' if alert.resolved?
  return resolve_button(alert)
end

def resolve_button(alert)
  button_to('Resolve', resolve_alert_path(alert), method: :put, class: 'button small danger dark')
end
```